server/middleware/levels.coffee

```diff
@@ -124,6 +125,18 @@ module.exports =
         level.get('adventurer'),
         req.features.codePlay and codePlay.canPlay(level.get('slug'))
       ])
+
+      if req.query.campaign and not canPlayAnyway
+        # check if the campaign requesting this is game dev hoc, if so then let it work
+        query = {
+          _id: mongoose.Types.ObjectId(req.query.campaign),
+          slug: 'game-dev-hoc', # replace with a way of indicating that 
+        }
+        query['levels.'+level.get('original')] = {$exists: true}
```

Comment:
Why is this a separate line?

Reply:
Cause dynamic keys + object literals don't mix, or so I thought. Can't in JS, but looks like possible in CS. Tweaked it.